R/rd.r

```diff
@@ -8,10 +8,10 @@
 #' @param package package name, as character vector
 #' @export
 #' @examples
-#' parse_rd("whisker.render", "whisker")
+#' # parse_rd("whisker.render", "whisker")
```

Comment:
It tells me

Fehler in mypaste(...) : Need character strings

could not make sense of that so far...